server/src/main/scala/ch.epfl.scala.index.server/Server.scala

```diff
@@ -60,7 +62,13 @@ object Server {
         dependencies <- api.dependencies()
         latestProjects <- api.latestProjects()
         latestReleases <- api.latestReleases()
-      } yield views.html.frontpage(keywords, targets, dependencies, latestProjects, latestReleases, userInfo)
```

Comment:
There's no need to do so :smile:. Let's just merge this and will take care of scalafmt afterwards. The vertical aligning option is not working as expected in some parts of the code, that's why I'm putting it off.